pages/infrastructure_providers-validators/how_to_set_up_full_node.md

```diff
@@ -1,65 +1,125 @@
+The following steps will guide you through manually setting up a full node.
+
+Run the commands in this procedure from your home directory unless otherwise specified. To change directories to your home folder, run the following command:
+```bash
+cd $HOME
+```
+
+### Step 1: Update your system and prepare to install dependencies
+To download system updates and install [curl](https://curl.se/),[jq](https://jqlang.github.io/jq/), and [lz4](https://lz4.org/), run the following commands:
 ```bash
 sudo apt-get -y update
 sudo apt-get install -y curl jq lz4
 ```
 
 ### Step 2: Install Go
-Install Go and set the environment variables.  This allows cosmovisor to be installed. amd64 architecture is used in this example.
+To install [Go](https://go.dev/), run the following commands using the latest version of Go:
+
+```bash
+# Example for AMD64 architecture and Go version 1.22.2
+wget https://golang.org/dl/go1.22.2.linux-amd64.tar.gz # Download the compressed file
+sudo tar -C /usr/local -xzf go1.22.2.linux-amd64.tar.gz # Extract the file to /usr/local
+rm go1.22.2.linux-amd64.tar.gz # Delete the installer package
+```
+
+Add the Go directory to your system `$PATH`:
+```bash
+echo 'export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin' >> $HOME/.bashrc # Write to your .bashrc profile
+```
+
+### Step 3: Install Cosmovisor and create data directories
+[Cosmovisor](https://docs.cosmos.network/main/build/tooling/cosmovisor) is a process manager for Cosmos SDK-based blockchains that enables automatic binary updates without downtime. To install the latest version of Cosmovisor, run the following command:
 ```bash
-wget https://golang.org/dl/go1.22.2.linux-amd64.tar.gz
-sudo tar -C /usr/local -xzf go1.22.2.linux-amd64.tar.gz
-rm go1.22.2.linux-amd64.tar.gz
-echo 'export PATH=$PATH:/usr/local/go/bin:$HOME/go/bin' >> $HOME/.bashrc
-eval "$(cat $HOME/.bashrc | tail -n +10)"
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@latest
 ```
 
-### Step 3: Install Cosmovisor
-Cosmovisor is a process manager for Cosmos SDK-based blockchains. It allows for the automatic updating of the binary without downtime.  The following command installs Cosmovisor.
+To create data directories for Cosmovisor, run the following commands:
 ```bash
-go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0
+mkdir -p $HOME/.dydxprotocol/cosmovisor/genesis/bin
+mkdir -p $HOME/.dydxprotocol/cosmovisor/upgrades
 ```
 
-### Step 4: Download dydxprotocold Binaries and Initialize Node
-1. From https://github.com/dydxprotocol/v4-chain/releases/ | Look for the `protocol` assets.
+### Step 4: Download the `dydxprotocold` binary
+The `dydxprotocold` binary contains the software you need to operate a full node. **You must use the same version of the software as the network to which you want to connect.**
+> To find the current version of the [dYdX foundation](https://www.dydx.foundation/) mainnet, see the recommended protocol version on [mintscan.io](https://www.mintscan.io/dydx/parameters).
```

Comment:
nice! thanks for highlighting this

nit: does this work? didnt get to test if this works
```suggestion
[!IMPORTANT] To find the current version of the [dYdX foundation](https://www.dydx.foundation/) mainnet, see the recommended protocol version on [mintscan.io](https://www.mintscan.io/dydx/parameters).
```

Reply:
no, I havent gotten any sort of highlight/callout to work besides the default note. would definitely like to, whether theres some way with this markdown or its something we can add later